api/v1alpha1/agentconfig_types.go

```diff
@@ -0,0 +1,45 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type AgentConfigurationSpec struct {
+	ClusterPingInterval               *string `json:"clusterPingInterval,omitempty"`
+	RuntimeServicesPingInterval       *string `json:"runtimeServicesPingInterval,omitempty"`
+	StackPollInterval                 *string `json:"stackPollInterval,omitempty"`
+	PipelineGateInterval              *string `json:"pipelineGateInterval,omitempty"`
```

Comment:
We should potentially have a way to disable these as well

Reply:
I've added such a possibility when you set the interval to "0s"

Reply:
I had to update the poller to support it. There is also the unit test for it